frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/dateProperty.component.tsx

```diff
@@ -16,27 +16,17 @@
  */
 
 import { PropertyDefinition } from '@/v5/store/tickets/tickets.types';
-import { DateTimePicker } from '@mui/lab';
-import { TextField } from '@mui/material';
+import { FormDatePicker } from '@controls/formDatePicker/formDatePicker.component';
 
-export const DateProperty = ({ property, value }: {property: PropertyDefinition, value:any}) => {
-	const maskValue = (val) => (!val ? '' : new Date(val));
-	return (
-		<div>
-    &nbsp;
-			<DateTimePicker
-				label={property.name}
-				inputFormat="MM/DD/YYYY"
-				value={maskValue(value)}
-				onChange={() => { }}
-				renderInput={({ error, ...params }) => (
-					<TextField
-						{...params}
-						value={maskValue(value)}
-						disabled={property.readOnly}
-					/>
-				)}
-			/>
-		</div>
-	);
-};
+export const DateProperty = ({
+	property: { name, readOnly, required },
+	value,
+}: { property: PropertyDefinition, value: any }) => (
+	<FormDatePicker
+		label={name}
+		name={name}
+		disabled={readOnly}
+		defaultValue={value}
+		reuiqred={required}
+	/>
```

Comment:
Does this need to be a separate component as it's so simple? Or is it neater to have all the components in the /properties/ folder?

Reply:
This as an `adpater`, something that takes a property and a value and passes it down to the actual input. It's there for simplicity as you do not want to use a switch to decide how to modify the property for each ticket input.
Instead, there is a single entry point for all the properties that acts as an interface (that all the `adapter`s adhere to) 
![image](https://user-images.githubusercontent.com/43547703/197207429-9d817c02-81ca-4cfc-8d3d-b54e1b13925f.png)

But let's see what @sanmont3drepo says about it